api/fileProcessing/index.js

```diff
@@ -0,0 +1,113 @@
+var models = require('../../models');
+var log = require('../../logging');
+var randomstring = require("randomstring");
+
+module.exports = function(app) {
+  var apiUrl = '/api/fileProcessing';
+
+  /**
+   * @api {get} /api/fileProcessing Get a new file processing job
+   * @apiName getNewFileProcessingJob
+   * @apiGroup FileProcessing
+   *
+   * @apiParam {String} type Type of recording.
+   * @apiParam {String} state Processing state.
+   */
```

Comment:
If the file processing API is internal only, perhaps it shouldn't be included in the docs?

Reply:
The current code for generating the docs doesn't include the fileProcessing directory so it won't be included. But I included the documentation style commenting for self reference.

Reply:
Ok, that's fair enough